platform/brains2/src/lib/usb/usb_descriptors.c

```diff
@@ -91,13 +91,17 @@ uint8_t const *tud_descriptor_configuration_cb(uint8_t index) {
 //--------------------------------------------------------------------+
 
 // array of pointer to string descriptors
-char const *string_desc_arr[3] = {
+char const *string_desc_arr[7] = {
     (const char[]){0x09, 0x04}, // 0: is supported language is English (0x0409)
     "Dato",                     // 1: Manufacturer
-    "Dato DUO"                  // 2: Product
-};
+    "Dato DUO",                 // 2: Product
+    NULL,                       // 3: Serial. Built upon request.
+    GIT_TAG,
+    GIT_BRANCH,
+    GIT_HASH};
```

Comment:
These string descriptors will not be reported by the OS by default, but we can query for them directly.
I will add a small python script for doing this.